.env.example

```diff
@@ -17,15 +17,24 @@ VITE_POLYGON_ZK_EVM_NETWORK_ID=1
 VITE_BRIDGE_API_URL=http://localhost:8080
 
 # FIAT EXCHANGE RATES API
-VITE_USE_FIAT_EXCHANGE_RATES=true
+VITE_ENABLE_FIAT_EXCHANGE_RATES=true
 VITE_FIAT_EXCHANGE_RATES_API_URL=https://api.exchangeratesapi.io/v1/latest
 VITE_FIAT_EXCHANGE_RATES_API_KEY=
 VITE_FIAT_EXCHANGE_RATES_ETHEREUM_USDC_ADDRESS=0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48
 
-# MISC
+# OUTDATED NETWORK MODAL
 VITE_SHOW_OUTDATED_NETWORK_MODAL=true
 VITE_OUTDATED_NETWORK_MODAL_TITLE=Welcome to the new Polygon zkEVM testnet!
 VITE_OUTDATED_NETWORK_MODAL_MESSAGE_PARAGRAPH_1=To connect, follow the instructions available at this site. 
 VITE_OUTDATED_NETWORK_MODAL_MESSAGE_PARAGRAPH_2=The previous network is still available until January 5th. You can access using the information available at:
 VITE_OUTDATED_NETWORK_MODAL_URL=https://deprecated.zkevm-test.net
+
+# DEPOSIT WARNING
 VITE_ENABLE_DEPOSIT_WARNING=true
+
+# REPORT FORM
+VITE_ENABLE_REPORT_FORM=true
+VITE_REPORT_FORM_URL=https://docs.google.com/forms/d/xxxxxx/viewform
+VITE_REPORT_FORM_ERROR_ENTRY=entry.xxxxxx
+VITE_REPORT_FORM_PLATFORM_ENTRY=entry.xxxxxx
+VITE_REPORT_FORM_URL_ENTRY=entry.xxxxxx
```

Comment:
All these renames and new variables must be also updated in both `scripts/deploy.sh` and the `README.md`.

Reply:
Added!